### PR TITLE
IGAPP-809: Languages json not existing

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

~~This PR probably does not fix anything and is rather meant to gather additional information. This bug should not be critical as I tried and afaics all errors logged are caught and handled correctly.~~

~~The error that the `languages.json does not exist` always occurs after another error during fetching, most of the time after the following error: `Failed to load the request for the languages endpoint. Body not allowed for GET or HEAD requests` (I also found [one occurrence](https://sentry.tuerantuer.org/organizations/digitalfabrik/issues/264/events/25db970a57814cc89d608f53896a1686/?project=2) of the same error for the cities endpoint, so it should not be the fault of the language endpoint). Nearly always it also seems like `loadCityContent` (or `loadCities` for the one occurrence) are called and run twice.~~

~~The whole thing is really weird, first because of the iniital GET/Body request thingy, second because the following `languages.json does not exist` error only occurs after this error (and not e.g. after regular fetch errors) and third because sagas are run twice even though we are using `takeLatest` which should cancel any previous sagas. I suspect that the issue is somewhere in executing `loadLanguages` twice though.~~

See below :)

### CODE REVIEW

**Comment by klinzo** [comment]
Finally found the issue :hear_no_evil: `merge´ also alters the first parameter so whenever a post request with body was made all following get requests had the same body set which lead to errors oc

File: .circleci/config.yml [hunk 1]
```diff
@@ -517,6 +517,9 @@ jobs:
             - run:
                 command: yarn lint
                 name: Lint
+            - run:
+                command: yarn workspace web stylelint
+                name: Stylelint
```
Comment (SteffiStuffel) [inline .circleci/config.yml:522]: Is style-linting not available for native?

File: api-client/src/Endpoint.ts [hunk 2]
```diff
@@ -61,7 +61,9 @@ class Endpoint<P, T> {
     const response = await fetch(url, requestOptions).catch((e: Error) => {
       throw new FetchError({
         endpointName: this.stateName,
-        innerError: e
+        innerError: e,
+        url,
```
Comment (SteffiStuffel) [inline api-client/src/Endpoint.ts:65]: Can you explain why it is necessary to add this params or how they fix the error?

**Review by SteffiStuffel — APPROVED** [review]
Not tested, as I do not know how to reproduce.

File: api-client/src/Endpoint.ts (hunk 2 above)
Comment (klinzo) [reply api-client/src/Endpoint.ts:65]: Just adds more information to debug, not related to fixing the actual error.

File: .circleci/config.yml (hunk 1 above)
Comment (klinzo) [reply .circleci/config.yml:522]: No idea, at least we don't use it on native.

**Review by Taggotty — APPROVED** [review]

**Pull request merged**